### PR TITLE
Auto Set Time Fields

### PR DESCRIPTION
**Pull request by JandenMa**
## based on `master` branch

- added global parameter `globalAutoSetTimeFields` to set default fields like `updatedAt`
- added parameter `autoSetTimeFields` in model constructor to set them for whole the model
- added checking logic for autoSetTimeFields in update
- optimized something

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,14 +172,15 @@ _This library is built for who uses GraphQL on NodeJS, you can use model to oper
...
 
-    | Key           | Type     | Introduction                       | Default value |
-    | ------------- | -------- | ---------------------------------- | ------------- |
-    | host          | `string` | Postgresql server host             | "localhost"   |
-    | port          | `number` | Postgresql server port             | 5432          |
-    | userName      | `string` | Postgresql server user name        | "postgres"    |
-    | password      | `string` | Postgresql server password         | ""_(empty)_   |
-    | database      | `string` | Postgresql database name           | "postgres"    |
-    | connectionMax | `number` | Postgresql database max connection | 10            |
+    | Key                     | Type            | Introduction                                                 | Default value |
```
Comment (Demerjianm) [inline README.md:176]: table seems to got all messed up agaib

Comment (JandenMa) [reply README.md:176]: I think no bad just something too long. Could you pull it to you local and try it?

**Review by JakeTompkins — APPROVED** [review]
Looks good to me. Let's see if it breaks!

**Comment by JandenMa** [comment]
> Looks good to me. Let's see if it breaks!

Try it, if fine I will publish a new version. thx

**Pull request merged**